src/MSALWrapper/AuthFlow/DeviceCode.cs

```diff
@@ -31,7 +31,7 @@ public class DeviceCode : IAuthFlow
         /// <summary>
         /// The silent auth timeout.
         /// </summary>
-        private TimeSpan silentAuthTimeout = TimeSpan.FromMinutes(5);
+        private TimeSpan silentAuthTimeout = TimeSpan.FromSeconds(15);

```

Comment:
Why do we have different timeout for each flow?

Reply:
AFAIK, Kyle might have used telemetry data for each silent `authflow` and used the numbers accordingly